packages/arb-token-bridge-ui/src/hooks/CCTP/useUpdateUSDCBalances.ts

```diff
@@ -5,21 +5,37 @@ import { useBalance } from '../useBalance'
 import { useNetworks } from '../useNetworks'
 import { useNetworksRelationship } from '../useNetworksRelationship'
 import { Address } from '../../util/AddressUtils'
+import { isNetwork } from '../../util/networks'
+
+function getL1AddressFromAddress(
+  address: string | undefined,
+  childChainId: number
```

Comment:
We pass `childChainId` to check for 2 things:

1. If it's an Orbit chain. If yes, we want to return L2 USDC address as the parent address. Otherwise L1 address.
2. If we're on a testnet. BalanceUpdater won't pass any USDC addresses so we need to default. We know whether we are on an Orbit chain and/or a testnet, so we can safely default the correct address.